backends/Base.h

```diff
@@ -308,7 +316,10 @@ class Base
 		struct stat statBuffer;
 		if (m_rank == 0 && stat(file.c_str(), &statBuffer) == 0) {
 			logWarning() << file << "already exists. Creating backup.";
-			rename(file.c_str(), (file + ".bak_" + utils::TimeUtils::timeAsString("%F_%T", time(0L))).c_str());
+			if (!m_backupTimeStamp.has_value()) {
```

Comment:
this should in theory never happen (for seissol), as m_backupTimeStamp is initialized in src/main.cpp?

Reply:
> this should in theory never happen (for seissol), as m_backupTimeStamp is initialized in src/main.cpp?

In the case of SeisSol, I totally agree. `XdmfWriter` can be used for other projects or sub-project where people do not care about having a unique timestamp for the entire application

Reply:
ok